### DISCUSSION TITLE
ハイパーリファレンス

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1849] [Q&A]
\userpackage{hyperref} を使おうとしていますが、うまくいきません。 re:view で必要な間違ったフォーマットはありますか?

どうもありがとうございます！

![image](https://user-images.githubusercontent.com/49851974/182495201-8175151d-b0de-42ab-a39f-18ab0f518f48.png)
![image](https://user-images.githubusercontent.com/49851974/182495234-d217c632-abac-4770-a7e5-b7ed2109404a.png)
![image](https://user-images.githubusercontent.com/49851974/182495289-a1e30bd3-e4d8-4cb2-9c08-0e65c7fdab36.png)

**Comment by kmuto** [comment]
あなたがRe:VIEWと無関係のoff-topicな話題を続けていることに警告します。

その上で、
- hyperrefパッケージはすでにロードされており、review-customで宣言する必要はありません。
- 例示しているrefが何を参照しようとしているのかを想像してください。refは、labelで宣言された箇所から章・節・項・図表番号といった「番号」を参照します。サンプルのsec:hyperrefのlabelの箇所では番号が何も紐付かないため、refの結果は空になります。
- たとえば `\pageref` を使えばページ参照となり、結果が得られるでしょう。
